src/manifest.safari.json

```diff
@@ -245,6 +244,7 @@
         "*://*.google.ng/*"
       ],
       "run_at": "document_start",
+      "all_frames": true,
```

Comment:
In the current form, I also think it does not need to run on all frames.

(We rely on blocklists to deal with the "ping" and beacon requests. In the original version there was logic to remove the ping attributes and patch the beacon API; in that case, it might be needed. But now, we only rely on rewriting the URL when it is being clicked. For that, I don't think iframes are relevant.)